torch_object_detection_segmenter.py

```diff
@@ -186,9 +186,13 @@ def _crop_image(img, target_size: Union[Tuple[int, int], int], top: int = None,
                     logging.debug(
                         f'detected {label_name} with confidence {score} at position {(top, left)} and size {target_size}')
                     batched.append(
-                        dict(offset=0, weight=1., blob=_img,
-                             location=(top, left), tags={'label': label_name}))
+                        Document(dict(offset=0, weight=1., blob=_img,
+                             location=[top, left], tags={'label': label_name})))
+
+            #result.append(batched)
+            # create a chunk for each of the objects detected for each image
 
-            result.append(batched)
+            docs[i].chunks = batched
+            i += 1
```

Comment:
this wont work with batching

Reply:
My latest commit works ?

Reply:
batching is wrong, u are not batching, so that is why it looks like working. But will not work if u pass we can say a DocumentArray of 1000 images